bos

```diff
@@ -1051,28 +1051,34 @@ prog
   .option('--external-funding', 'Use external funds for the channel open')
   .option('--give <give_amount>', 'Amount to gift to peer', REPEATABLE)
   .option('--node <node_name>', 'Node to open channels')
+  .option('--opening-node <opening_node>', 'Node to open channels', REPEATABLE)
   .option('--set-fee-rate <ppm>', 'Set forward fee rate to peer', REPEATABLE)
   .option('--type <type>', 'Type of channel (private/public)', REPEATABLE)
   .action((args, options, logger) => {
     return new Promise(async (resolve, reject) => {
       const collect = n => flatten([n]).filter(n => !!n);
+      const openingNodes = n => flatten([n]);
+      const nodes = !!options.openingNode ? openingNodes(options.openingNode).map(String) : [];
+      const lnds = !!nodes.length ? await Promise.all(nodes.map(async node => (await lnd.authenticatedLnd({logger, node: node !== 'true' ? node: options.node})).lnd)) : [];
 
       try {
-        return peers.openChannels({
-          logger,
-          ask: (n, cbk) => inquirer.prompt([n]).then(res => cbk(res)),
-          capacities: flatten([options.amount].filter(n => !!n)),
-          cooperative_close_addresses: collect(options.coopCloseAddress),
-          fs: {getFile: readFile},
-          gives: flatten([options.give].filter(n => !!n)),
-          is_external: options.externalFunding,
-          lnd: (await lnd.authenticatedLnd({logger, node: options.node})).lnd,
-          public_keys: args.peerPublicKeys,
-          request: commands.simpleRequest,
-          set_fee_rates: flatten([options.setFeeRate]).filter(n => !!n),
-          types: flatten([options.type].filter(n => !!n)),
-        },
-        responses.returnObject({logger, reject, resolve}));
+          return peers.openChannels({
```

Comment:
I think there is an indenting issue here that makes it look like these lines were changed